.github/workflows/release.yml

```diff
@@ -17,6 +17,33 @@ jobs:
       - uses: actions/checkout@v3
       - id: go-version
         run: echo "::set-output name=version::$(cat ./.go-version)"
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Batch changes
+          uses: miniscruff/changie-action@v0
+          with:
+            version: latest
+            args: batch ${{ github.ref_name }}
```

Comment:
Nit: `github.ref_name` will contain the `v` from the version tag that triggered the workflow, which doesn't match the current CHANGELOG format. There's ways to remove that `v` in this current setup but other changes to this workflow could alleviate the need to worry about this.